rustls-platform-verifier/src/lib.rs

```diff
@@ -49,17 +51,14 @@ pub use tests::ffi::*;
 /// **Important:** You must ensure that your `reqwest` version is using the same Rustls
 /// version as this crate or it will panic when downcasting the `&dyn Any` verifier.
 ///
-/// If you require more control over the rustls `ClientConfig`, you can
-/// instantiate a [Verifier] with [Verifier::default] and then use it
-/// with [`DangerousClientConfigBuilder::with_custom_certificate_verifier`][rustls::client::danger::DangerousClientConfigBuilder::with_custom_certificate_verifier].
+/// If you require more control over the rustls [`ClientConfig`], you can import the
+/// [`BuilderVerifierExt`] trait and call `.with_platform_verifier()` on the [`ConfigBuilder`].
 ///
 /// Refer to the crate level documentation to see what platforms
 /// are currently supported.
+#[deprecated(since = "0.4.0", note = "use the `ConfigVerifierExt` instead")]
```

Comment:
Would you mind updating the deprecated functions described in the README's [Installation and setup section](https://github.com/rustls/rustls-platform-verifier/blob/6927b45f354b780f4a5409f4c070cd246c9703b1/README.md?plain=1#L85-L115)? I think there's both a `tls_config()` and a `tls_config_with_provider()` invocation.